x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/index.tsx

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { some } from 'lodash/fp';
+import { EuiFlyoutHeader, EuiFlyoutBody, EuiSpacer } from '@elastic/eui';
+import React, { useMemo } from 'react';
+import styled from 'styled-components';
+import deepEqual from 'fast-deep-equal';
+
+import { BrowserFields, DocValueFields } from '../../../../common/containers/source';
+import { ExpandableEvent, ExpandableEventTitle } from './expandable_event';
+import { useTimelineEventsDetails } from '../../../containers/details';
+import { TimelineTabs } from '../../../../../common/types/timeline';
+
+const StyledEuiFlyoutBody = styled(EuiFlyoutBody)`
+  .euiFlyoutBody__overflow {
+    display: flex;
+    flex: 1;
+    overflow: hidden;
+
+    .euiFlyoutBody__overflowContent {
+      flex: 1;
+      overflow: hidden;
+      padding: ${({ theme }) => `${theme.eui.paddingSizes.xs} ${theme.eui.paddingSizes.m} 64px`};
+    }
+  }
+`;
+
+interface EventDetailsPanelProps {
+  browserFields: BrowserFields;
+  docValueFields: DocValueFields[];
+  expandedEvent: { eventId: string; indexName: string };
+  handleOnEventClosed: () => void;
+  isFlyoutView?: boolean;
+  tabType: TimelineTabs;
+  timelineId: string;
+}
+
+const EventDetailsPanelComponent: React.FC<EventDetailsPanelProps> = ({
+  browserFields,
+  docValueFields,
+  expandedEvent,
+  handleOnEventClosed,
+  isFlyoutView,
+  tabType,
+  timelineId,
+}) => {
+  const [loading, detailsData] = useTimelineEventsDetails({
+    docValueFields,
+    indexName: expandedEvent.indexName ?? '',
+    eventId: expandedEvent.eventId ?? '',
+    skip: !expandedEvent.eventId,
+  });
+
+  const isAlert = useMemo(
```

Comment:
I do not think that we need to do a useMemo since isAlert is a primitive and `ExpandableEvent` component should be able to manage it.

Reply:
Yea, you're right. Removed the `useMemo`